application/helpers/admin/import_helper.php

```diff
@@ -1285,6 +1285,9 @@ function XMLImportSurvey($sFullFilePath, $sXMLdata = null, $sNewSurveyName = nul
         if ($newSurvey->sid) {
             $iNewSID = $results['newsid'] = $newSurvey->sid;
             $results['surveys']++;
+            if (!empty($iDesiredSurveyId) && $iNewSID != $iDesiredSurveyId) {
+                $results['importwarnings'][] = gT("The desired survey ID was invalid and could not be assigned to the new survey.");
```

Comment:
clear enough

Reply:
"The desired survey ID was already in use, therefore a random one was assigned."